actions/global_actions.jsx

```diff
@@ -492,21 +501,23 @@ export async function redirectUserToDefaultTeam() {
         }
     }
 
-    if (teams[teamId]) {
+    const team = teams[teamId];
+    if (team) {
         const channelId = BrowserStore.getGlobalItem(teamId);
         const channel = ChannelStore.getChannelById(channelId);
-        if (channel) {
-            redirect(teams[teamId].name, channel.name);
+        let channelName = Constants.DEFAULT_CHANNEL;
+        if (channel && channel.team_id === team.id) {
+            selectChannel(channel.id)(dispatch, getState);
```

Comment:
Can you do dispatch(selectChannel(channel.id));

Reply:
Oopss, sure. Now updated, thanks!